packages/xo-server-cloud/src/index.js

```diff
@@ -76,6 +86,16 @@ class XoServerCloud {
     return catalog
   }
 
+  async _getAllCatalog() {
```

Comment:
the initial function that was not written by me is : `getCatalog` which returns Catalog with only registred XVAs.
On the other hand `getAllCatalog` returns all free XVAs which do not require specific registration from `www-xo`
`_getWholeCatalog` is good, maybe `_getFreeCatalog` is better. @julien-f

Reply:
What's the difference between both implementations?

Reply:
`getCatalog` which returns Catalog with only registred XVAs.
On the other hand `getAllCatalog` returns all free XVAs which do not require specific registration from `www-xo`

Reply:
`getAllCatalog` is incorrect because there is only one catalog, `getWholeCatalog` is incorrect because it does not return the whole catalog but only resources which does not need to be registered.

Reply:
I agree. I think `getFreeCatalog` is more relevant.

Reply:
IMHO, *free* meaning is unclear, I'd love to see `www-xo` documentation on this.